src/Logary/Internals/NullLogger.fs

```diff
@@ -8,7 +8,7 @@ open Logary
 /// itself.
 type private NullLogger() =
   interface Logger with
-    member x.logWithAck (_, _) _ = LogResult.success
```

Comment:
Why move this? Now it's a three-object allocation every call, as opposed to re-using from the deleted module above, which doesn't allocate anything.

Reply:
you are right, this can be avoid, this change is because `LogResultModule` is in a lower file order, `NullLogger` can not reference.